airflow/config_templates/config.yml

```diff
@@ -1349,6 +1349,13 @@ operators:
 webserver:
   description: ~
   options:
+    access_denied_message:
+      description: |
+        The message displayed when an user tries to perform operations outside their rights.
+      version_added: 2.7.0
+      type: string
+      example: ~
+      default: "Access is Denied"
```

Comment:
```suggestion
      default: "Unauthorized Access: Denied"
```

Reply:
I dont think we can do that. The current message is "Access is Denied" so in order to keep the current experience, we got to have this default value

Reply:
Agreed. We do not want to break the backward compatibility experience. If we want to make this change, we should have a different task for it instead

Reply:
I agree with @vincbeck